Transform array of enums in multi-options

NoxTav opened the pull request:
## Description

Currently, when we have an array of enum, we have to fill it in JSON, but if there is an enum, we can select a possibility in an option.

There is the possibility of selecting multiple values of an enum with the multi-options.

## Effects in n8n UI

Can select the values in fixed mode in the list.

## Side effects

Someone wanting to have several instances of the same value would not be able to add them directly, but they can pass in expression mode and update them.

Commit:
transform array of enums in multi-options

devlikepro commented on src/n8n/SchemaToINodeProperties.ts:
```diff
@@ -43,6 +43,8 @@ export class N8NINodeProperties {
         let type: NodePropertyTypes;
         let defaultValue = this.schemaExample.extractExample(schema)
 
+        let options = undefined
```
Is there any type for `options` available from n8n ?

NoxTav replied:
I added the type and the test.

devlikepro reviewed (commented):
could you add one small test pls :pray:

NoxTav commented:
Hi @devlikepro, 
Just a friendly ping — I’ve addressed the feedback and updated the PR. When you have a moment, could you please take another look? Your review would be much appreciated. Thanks!

NoxTav commented:
Hi @devlikepro,
Did you have any chance to look at the issue?
